app/listeners/webhook_listener.rb

```diff
@@ -101,6 +101,7 @@ def handle_typing_status(event_name, event)
     payload = {
       event: event_name,
       user: user.webhook_data,
+      inbox_id: inbox.id,
       conversation: conversation.webhook_data,
```

Comment:
It should also be noted the `inbox_updated` event (which does not seem to be in use right now) does not include the inbox id in the payload, so this new logic will not apply if `inbox_update` ever comes into play.

Should we mess with the `inbox_update` payload to ensure future compatibility, or maybe just add a comment somewhere?